DESCRIPTION

```diff
@@ -30,7 +30,8 @@ Imports:
 Suggests:
     knitr,
     rmarkdown,
-    testthat (>= 2.1.0)
+    testthat (>= 3.0.3),
+    vdiffr (>= 1.0.0)
```

Comment:
_vdiffr_ needs _testthat_ version `> 3.0.3`, so we also need to bump accordingly.

Reply:
Actually what happens if a package listed in **Suggests:** is installed in a lower version than required? 
Does the user get a warning then?

Reply:
No. Users actually don't install anything from `Suggests`. So this is mostly CI/CD-facing. 

They won't be able to install _vdiffr_ if they don't have the right version of _testthat_, and won't be able to test runs locally if they are on a platform that ships tests with the package.